src/vfocus.js

```diff
@@ -315,3 +315,7 @@ VFocus.prototype.indexOf = function(focus){
 VFocus.prototype.spliceChildren = function(){
   return Array.prototype.splice.apply(this.children(), arguments);
 };
+
+VFocus.prototype.getChildAt = function(index){
+  return this.vNode.children[index];
```

Comment:
I think the method name makes this a bit confusing, since you call it on a `vFocus` but get a `vNode` returned. Could we return the child as a `vFocus` instead?